packages/nvidia_nat_atif/src/nat/atif/scripts/export_trajectory_to_phoenix/README.md

```
@@ -0,0 +1,128 @@
+```
+ATIF Trajectory (dict)
+  -> ATIFTrajectorySpanExporter.convert()  -> list[Span]
+  -> convert_spans_to_otel_batch()         -> list[OtelSpan]
+  -> HTTPSpanExporter.export()             -> Phoenix
+```
+
+### Span hierarchy
+
+Each trajectory produces the following span tree:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)